RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -43,41 +44,57 @@ DAClusterizerInZT_vect::DAClusterizerInZT_vect(const edm::ParameterSet& conf) {
-    std::cout << "DAClusterizerinZT_vect: tmerge = " << tmerge_ << std::endl;
-    std::cout << "DAClusterizerinZT_vect: Tmin = " << minT << std::endl;
-    std::cout << "DAClusterizerinZT_vect: Tpurge = " << purgeT << std::endl;
-    std::cout << "DAClusterizerinZT_vect: Tstop = " << stopT << std::endl;
-    std::cout << "DAClusterizerinZT_vect: vertexSize = " << vertexSize_ << std::endl;
-    std::cout << "DAClusterizerinZT_vect: vertexSizeTime = " << vertexSizeTime_ << std::endl;
-    std::cout << "DAClusterizerinZT_vect: coolingFactor = " << coolingFactor_ << std::endl;
-    std::cout << "DAClusterizerinZT_vect: d0CutOff = " << d0CutOff_ << std::endl;
-    std::cout << "DAClusterizerinZT_vect: dzCutOff = " << dzCutOff_ << std::endl;
-    std::cout << "DAClusterizerinZT_vect: dtCutoff = " << dtCutOff_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: mintrkweight = " << mintrkweight_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: uniquetrkweight = " << uniquetrkweight_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: zmerge = " << zmerge_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: tmerge = " << tmerge_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: Tmin = " << minT << std::endl;
+    std::cout << "DAClusterizerInZT_vect: Tpurge = " << purgeT << std::endl;
+    std::cout << "DAClusterizerInZT_vect: Tstop = " << stopT << std::endl;
+    std::cout << "DAClusterizerInZT_vect: vertexSize = " << vertexSize_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: vertexSizeTime = " << vertexSizeTime_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: coolingFactor = " << coolingFactor_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: d0CutOff = " << d0CutOff_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: dzCutOff = " << dzCutOff_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: dtCutoff = " << dtCutOff_ << std::endl;
+    std::cout << "DAClusterizerInZT_vect: zrange = " << sel_zrange_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: convergence mode = " << convergence_mode_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: delta_highT = " << delta_highT_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: delta_lowT = " << delta_lowT_ << std::endl;
   }
+#ifdef DEBUG
+  std::cout << "DAClusterizerinZT_vect: DEBUGLEVEL " << DEBUGLEVEL << std::endl;
 #endif
 
+  if (convergence_mode_ > 1) {
+    edm::LogWarning("DAClusterizerinZT_vect")
+        << "DAClusterizerInZT_vect: invalid convergece_mode" << convergence_mode_ << "  reset do default " << 0;
+    convergence_mode_ = 0;
+  }
+
   if (minT == 0) {
-    edm::LogWarning("DAClusterizerinZT_vectorized")
-        << "DAClusterizerInZT: invalid Tmin" << minT << "  reset do default " << 1. / betamax_;
+    edm::LogWarning("DAClusterizerinZT_vect")
+        << "DAClusterizerInZT_vect: invalid Tmin" << minT << "  reset do default " << 1. / betamax_;
```

Comment:
Ok, very minor, but if you touch the code somewhere else then "reset do" -> "reset to" here

Reply:
done